addons/mail/models/models.py

```diff
@@ -105,6 +105,9 @@ def _mail_get_partner_fields(self, introspect_fields=False):
             ]
         return partner_fnames
 
+    def mail_get_partner_fields(self):
+        return self._mail_get_partner_fields(True)
```

Comment:
Would say don't introspect, if we don't have clear customer better skip timezone. Also always use named parameter for kwargs :) 
```suggestion
        return self._mail_get_partner_fields(introspect_fields=False)
```

Reply:
Sent `introspect_fields` as true, to cover all the cases like you suggested :sweat_smile: https://github.com/odoo/odoo/pull/178486#discussion_r1988971348
(case where studio is installed)

Reply:
Normally based code should check for it being present (see override in web_studio). Introspect fields is more a "hack" that was added for voip as far as I remember.